src/components/Filter/Filter.js

```diff
@@ -42,12 +42,12 @@ function Filter({filterShow, onToggleFilterShow, filterPrice}) {
 const FilterSection = styled.section`
     position: fixed;
     bottom: 50px;
-    width: 60vw;
-    height: 30vh;
+    width: 50vw;
+    height: 25vh;
```

Comment:
are you sure about setting this to vh? what if the user decides for landscape mode? How about not setting a height and use padding instead? Then it would be dynamic

Reply:
i changed